app/models/enterprise.rb

```diff
@@ -374,7 +374,7 @@ def geocode_address
   end
 
   def ensure_owner_is_manager
-    users << owner unless users.include?(owner) || owner.admin?
+    users << owner unless users.include?(owner)
```

Comment:
This looks good to me but I think now it's the time improve it and not rely on the callback. So, instead of having to check every single time an action is performed on the object, which triggers the `before_validation` callback I would rather do assign the owner as user of the enterprise at the same point in time where the enteprise is created.

for `app/controllers/api/enterprises_controller.rb` I see this happens in

https://github.com/openfoodfoundation/openfoodnetwork/blob/3399eba4281a4919d5de644e467b57b46c23f647/app/controllers/api/enterprises_controller.rb#L52

but I fail to see if we do that as well on other places such as `app/controllers/enterprises_controller.rb`. All this may possibly mean extracting some sort of `CreateEnterpriseService` and I really believe it would be a great move for the future.

Would you agree on creating and checking this out on a separate issue? this way we can benefit from this fix but improve right away.